dataingest/spatial/import.sh

```diff
@@ -16,7 +16,7 @@ tables["lad_gis"]="Local_Authority_Districts_(December_2017)_Boundaries_in_the_U
 for TABLE in "${!tables[@]}"; do
     GEOJSON="${tables[$TABLE]}"
     echo "creating '$TABLE' in '$PGDATABASE' on '$PGHOST'"
-    docker run $EXTRA -v $PWD:$PWD $DOCKER ogr2ogr -f "PostgreSQL" PG:"host=$PGHOST user=$PGUSER dbname=$PGDATABASE password=$PGPASSWORD port=$PGPORT" "$PWD/$GEOJSON" -nln "$TABLE" --config PG_USE_COPY YES -lco GEOM_TYPE=geometry
+    docker run $EXTRA -v $PWD:$PWD $DOCKER ogr2ogr -f "PostgreSQL" PG:"host=${PGHOST_INTERNAL:-$PGHOST} user=$PGUSER dbname=$PGDATABASE password=$PGPASSWORD port=${PGPORT_INTERNAL:-$PGPORT}" "$PWD/$GEOJSON" -nln "$TABLE" --config PG_USE_COPY YES -lco GEOM_TYPE=geometry
     psql -c "VACUUM ANALYZE $TABLE"
 done
 
```

Comment:
I need to check this on linux again but I don't think it works for me. But merge anyway and I can readd the old linux line if needed. This script is a priority (IMO) to rewrite in go anyway and has an existing ticket